Specimen/Specimen-BoneMarrowAspiration-Example.json

```diff
@@ -0,0 +1,44 @@
+{
+  "resourceType": "Specimen",
+  "id": "Specimen-BoneMarrowAspiration-Example",
+  "identifier": [
+    {
+      "system": "https://fhir.add.nhs.uk/Id/specimenId",
```

Comment:
Might need a different system as this number isn't being assigned from Addenbrookes. Maybe the local identifier OID? with the assigner being the SHIMDS lab at St. James?